exporter/opentelemetry-exporter-otlp-proto-grpc/tests/test_otlp_trace_exporter.py

```diff
@@ -275,21 +275,21 @@ def test_otlp_headers_from_env(self, mock_ssl_channel, mock_secure):
         exporter = OTLPSpanExporter()
         # pylint: disable=protected-access
         self.assertEqual(
-            exporter._headers, (("key1", "value1"), ("key2", "VALUE=2"))
+            exporter._headers, (("key1", "value1"),("key2", "VALUE=2"), ("user-agent", "OTel OTLP Exporter Python/1.13.0"))
```

Comment:
Don't hard-code the version part (1.13.0) because tests fail with a new version. Match the version dynamically by importing `__version__` or just check for the existence of the `user-agent` header name only.